src/mod_csi.erl

```diff
@@ -63,10 +63,12 @@ supported_features() ->
 %%% Hook handlers
 %%%
 
--spec c2s_stream_features([exml:element()], mongooseim:host_type(), jid:lserver()) ->
-          [exml:element()].
-c2s_stream_features(Acc, _HostType, _Lserver) ->
-    lists:keystore(<<"csi">>, #xmlel.name, Acc, csi()).
+-spec c2s_stream_features(Acc, Params, Extra) -> {ok, Acc} when
+    Acc :: [exml:element()],
+    Params :: map(),
+    Extra :: map().
```

Comment:
I think that we can use now in all these handlers the new type, `gen_hook:extra()`, just for good habit, but nevertheless it doesn't change much in this PR, just for the future 👌🏽